pkg/datafetcher/schema/atmos/manifest/1.0.json

```
@@ -2470,6 +2470,9 @@
             "stack": {
               "type": "string"
             },
+            "dependencies": {
```

Biome (2.5.6)

[error] The key dependencies was already declared.

(lint/suspicious/noDuplicateObjectKeys)